partition/roles/mgmt-server/tasks/install_frr.yaml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: ensure gpg is installed to handle FRR repo key
```

Comment:
It's a bit sad we have an FRR role now and the mgmt-server's FRR is still installed here without using it.